NextcloudTalk/NCChatViewController.m

```diff
@@ -436,6 +437,8 @@ - (void)viewWillAppear:(BOOL)animated
     
     [self checkRoomControlsAvailability];
     
+    [self startObservingExpiredMessages];
```

Comment:
This will add a timer each time the view gets added to the hierarchy. Problem is, this is called each time we enter the room infos and get back to the chat, resulting in multiple timers running:

```
2022-09-20 21:30:06.492256+0200 NextcloudTalk[17509:1155994] MM: removeExpiredMessages
2022-09-20 21:30:11.554399+0200 NextcloudTalk[17509:1155994] MM: removeExpiredMessages
2022-09-20 21:30:13.123294+0200 NextcloudTalk[17509:1155994] MM: removeExpiredMessages
2022-09-20 21:30:20.796434+0200 NextcloudTalk[17509:1155994] MM: removeExpiredMessages
```

<img width="567" alt="Bildschirmfoto 2022-09-20 um 21 30 55" src="https://user-images.githubusercontent.com/1580193/191347918-2e1d8a63-4d2c-470f-bfdf-3a4c10a78d37.png">

We should invalidate a potential existing timer before we add a new one or invalidate in `viewWillDisappear` for example.